sdk/resourcemanager/azure-resourcemanager-storage/src/main/java/com/azure/resourcemanager/storage/models/BlobServiceProperties.java

```diff
@@ -208,5 +222,25 @@ interface WithDeleteRetentionPolicy {
              */
             Update withDeleteRetentionPolicyDisabled();
         }
+
+        /** The stage of the blobserviceproperties update allowing to enable/disable blob versioning. */
+        interface WithBlobVersioning {
+            /**
+             * Enables blob versioning.
+             * @return the next update stage
+             */
+            Update withBlobVersioningEnabled();
+
+            /**
+             * Disables blob versioning.
+             * <p>After versioning is disabled, modifying the current version creates a blob that is not a version.
+             * All subsequent updates to the blob will overwrite its data without saving the previous state.
+             * All existing versions persist as previous versions.</p>
+             * <p>You can read or delete versions using the version ID after versioning is disabled.
+             * You can also list a blob's versions after versioning is disabled.</p>
```

Comment:
some of the sentence seems not well formed. e,g. `modifying the current version creates a blob that is not a version` (blob is certainly not a version...)

Reply:
OK. Didn't realize that when ctrl+c ...